google-startup-scripts/usr/share/google/get_metadata_value

```diff
@@ -14,9 +14,8 @@
 # limitations under the License.
 
 # Get a metadata value from the metadata server.
-
 declare -r VARNAME=$1
-declare -r MDS_PREFIX=http://169.254.169.254/computeMetadata/v1
+declare -r MDS_PREFIX=http://metadata/computeMetadata/v1
```

Comment:
We should not use the short form ```metadata```. If you want to switch away from the IP address, use ```metadata.google.internal```.